### PR TITLE
Support for Hachirumi

### PR DESCRIPTION
**Pull request by Curstantine**
Had to reopen a pull....(⌒_⌒;)

### CODE REVIEW

File: src/Hachirumi/Hachirumi.ts [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+
+    return createManga({
+      id: result.slug,
+      titles: [result.title],
+      image: HACHIRUMI_DOMAIN + result.cover,
+      rating: 0, // Rating is not supported by Hachirumi.
+      status: MangaStatus.ONGOING,
+      artist: result.artist,
+      author: result.author,
+      desc: result.description,
```
Comment (ConradWeiser) [inline src/Hachirumi/Hachirumi.ts:133]: Descriptions are read correctly, however they do not render all too well in the app. Take for example https://hachirumi.com/read/manga/love-cute-short-stories/

The description for this renders with a lot of HTML tags. `<a href=...>` `` etc.
These obviously don't render inside of the app.

I understand that the source doesn't really have many proper descriptions. I have a few suggestions:
- Either remove descriptions entirely and replace it with a static "Hachirumi does not provide descriptions"
- Parse out all of the HTML tags, grab the relevant information, and try to format it in a way which looks fine. Perhaps keep the HTML tag `text`, just get rid of the tags themselves?

**Review by ConradWeiser — CHANGES_REQUESTED** [review]
- Pages load kinda slowly though that's not really something you can do much about. I don't think it's enough to require a 'slow' tag on the source, so this is fine!
- Your descriptions should have some additional parsing work, see below

**Comment by ConradWeiser** [comment]
EFA28244-6811-483B-A2F0-09EAD909E1FF

example

**Review by ConradWeiser — APPROVED** [review]
Looks solid to me! Tests and works without issue

**Review by nar1n — APPROVED** [review]

**Pull request merged**